ukbb-rg/deployment.yaml

```diff
@@ -37,10 +37,10 @@ spec:
          imagePullPolicy: Always
          resources:
            requests:
-             memory: "3.75G"
-             cpu: "800m"
+             memory: 375Mi
```

Comment:
Too low?  Currently:

```
NAME                CPU(cores)   MEMORY(bytes)   
ukbb-rg-browser-0   1m           1657Mi          
ukbb-rg-static-0    0m           2454Mi          
```

Reply:
heh. I'm trying to have a set memory:cpu ratio to simplify how I think about resources. I just set the requests to 3750Mi / 1cpu. ukbb-rg is an unfortunate memory hog, I think we just accept that. It only runs once in default, so not so bad.